appstorereviews.js

```diff
@@ -1,13 +1,14 @@
 const controller = require('./reviews');
 const fs = require('fs');
 var request = require('request');
+var schedule = require('node-schedule');
 require('./constants');
 
 exports.startReview = function (config, first_run) {
 
-    if (config.regions === false){
+    if (config.regions === false) {
         try {
-            config.regions = JSON.parse(fs.readFileSync(__dirname  + '/regions.json'));
+            config.regions = JSON.parse(fs.readFileSync(__dirname + '/regions.json'));
```

Comment:
Maybe leaving the styling thing along will be better when implementing some functions? Otherwise the diff will including so many changes not related to the topic, make the review and code tracing a little bit harder.

It will be very great if the styling will be fixed with CI testing in another PR 👍

Reply:
I think so too. This PR undo unnecessary diffs 🙇